js/editor.js

```diff
@@ -100,4 +101,22 @@
     document.addEventListener('WebComponentsReady', function() {
         render(templateUtils.getTemplateOutput());
     });
+
+    /* Ensure that performance is supported before
+       gathering the performance metric */
+    if (performance !== undefined) {
+        document.addEventListener('readystatechange', function(event) {
```

Comment:
Sorry for the message on the old PR. 

Would it make more sense to listen for the [load event](https://developer.mozilla.org/en-US/docs/Web/Events/load) rather than the ``readystatechange`` event? This may avoid the 0.1 second wait for ``performance.timing.loadEventEnd`` to be populated.

Reply:
Definitely worth trying, thanks for the suggestion @jwhitlock